css/scss/_page-home.scss

```diff
@@ -0,0 +1,72 @@
+.sfh-wrapper {
+  margin: auto 0;
+  padding: 50px 50px;
```

Comment:
@praxoc , you can shorthand the padding value. It could be written more concisely as `50px` instead of `50px 50px`